test/hummingbot/connector/derivative/bybit_perpetual/test_bybit_perpetual_derivative.py

```diff
@@ -121,53 +170,18 @@ def test_create_buy_order(self, post_mock, nonce_provider_mock):
                                           price=Decimal("46000"),
                                           position_action=PositionAction.OPEN)
 
-        self.mocking_assistant.add_http_response(
-            post_mock,
-            200,
-            {
-                "ret_code": 0,
-                "ret_msg": "OK",
-                "ext_code": "",
-                "ext_info": "",
-                "result": {
-                    "user_id": 1,
-                    "order_id": "335fd977-e5a5-4781-b6d0-c772d5bfb95b",
-                    "symbol": "BTCUSD",
-                    "side": "Buy",
-                    "order_type": "Limit",
-                    "price": 8800,
-                    "qty": 1,
-                    "time_in_force": "GoodTillCancel",
-                    "order_status": "Created",
-                    "last_exec_time": 0,
-                    "last_exec_price": 0,
-                    "leaves_qty": 1,
-                    "cum_exec_qty": 0,
-                    "cum_exec_value": 0,
-                    "cum_exec_fee": 0,
-                    "reject_reason": "",
-                    "order_link_id": "",
-                    "created_at": "2019-11-30T11:03:43.452Z",
-                    "updated_at": "2019-11-30T11:03:43.455Z"
-                },
-                "time_now": "1575111823.458705",
-                "rate_limit_status": 98,
-                "rate_limit_reset_ms": 1580885703683,
-                "rate_limit": 100
-            })
-        buy_request_url, buy_request_headers, buy_request_data = asyncio.get_event_loop().run_until_complete(
-            self.mocking_assistant.next_sent_request_data(post_mock))
+        asyncio.get_event_loop().run_until_complete(asyncio.sleep(0.5))
```

Comment:
This would be a nice place to use the callback functionality and set an `asyncio.Event` that can be awaited here